src/Plugins/RunawayProcessKiller/RunawayProcessKillerExtension.cs

```diff
@@ -113,11 +116,12 @@ public override void OnWrapperStarted()
             else
             {
                 Logger.Warn("The process " + pid + " has no " + expectedEnvVarName + " environment variable defined. " 
-                    + "The process has not been started by this service, hence it won't be terminated.");
+                    + "The process has not been started by WinSW, hence it won't be terminated.");
```

Comment:
Isn't it `jenkins.exe` for Jenkins? Is it reasonable to expect users to know what WinSW is where this message will show up?